code/script.js

```diff
@@ -227,99 +270,298 @@ const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
+const clickCounter = () => {
+  count++;
+  guessCounter.innerHTML = `${count}`;
+}
+
+const startTimer = () => { // defining startTimer
+  clearInterval(timerIntervalId); // clears the interval before starting to count, in case of restarts
+  let second = 01,
+      minute = 10;
+
+  timerIntervalId = setInterval(() => {
+    timer.innerHTML = 
+    (minute < 10 ? "0" + minute + ":" : minute + ":") +  // if there are less then 10 minutes, add a 0 in front of the minute, else show the minutes
+    (second < 10 ? "0" + second + " remaining" : second + " remaining");  
+
+    second --;  //subtracts a second every second
+
+    if (second == 00){ //if seconds gets to 00, decrease minutes by one and return seconds to 59
+      minute--;
+      second = 59;
+    }
+    
+    if (minute == 0){
+      loseSound.play()
+      loseText.innerHTML = ''
+      loseText.innerHTML += `
+      GAME OVER.`
+      loseWrapper.style.display = 'flex'
+      board.style.display = 'none'
+      questionSection.style.display = 'none'
+      footer.style.display = 'none'
+    }
+
+  }, 1000);//for this function, it runs ever 1 second
+}
+
+
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
+
   // What else should happen when we start the game?
+  generateBoard();
+  setSecret(CHARACTERS);
+  startTimer();
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
```

Comment:
Okay so I tried to wrap my head around how we wrote the selectQuestion different, you needed to assign keep = false but I didn't. I don't fully grasp how the different ways works but here is how it looks in some code snippets from my file (let me know if you wanna inspect it or discuss it further). Interesting to see how different approaches works!

```
const checkQuestion = () => {
  const { category, value } = currentQuestion; // This is to destruct the currentQuestion object to be able to use these variables more easily
  if (category === 'eyes') {
    if (value === secret.eyes) {
      filterCharacters(keep);
    } else {
      filterCharacters();
    }
etc...
```
And the I passed on(?) the keep:

```
const filterCharacters = (keep) => {
  const { category, value } = currentQuestion

  if (category === 'accessories') {
    if (keep) {
      alert(
        `Yes, the person wears a ${value}! Keep all the people that wears a ${value}`);
        charactersInPlay = charactersInPlay.filter((person) => person[category].includes(value))
    } else {
      alert(
        `No, the person doesn't wear a ${value}! Remove all the people that wears a ${value}`)
        charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value))
    }
  }
etc...
```